### DISCUSSION TITLE
Android app

### DISCUSSION THREAD

**Discussion by kunsh13** [kunsh13/iptv#8] [General]
Try this android app for mobile phone and TV.

[IPTV.kunsh13.apk](https://github.com/kunsh13/iptv/blob/4892266d6bfc513f3f877266dbba7431b02157d6/_IPTV_13092023.apk)

Feeds are auto updated in apk. There is no need to install new apk every time when feeds are updated.

**Comment by tayyabios6** [comment]
i want to make web app link you.please contact with me  tayyabios6@gmail.com
